paddle/fluid/framework/ir/mkldnn/quant_dequant_mkldnn_pass.cc

```diff
@@ -445,11 +593,116 @@ void QuantDequantMkldnnPass::DequantizeOpWeights(
   weight_tensor->Resize(weight_dims);
 }
 
+void QuantDequantMkldnnPass::DequantizeOpWeightsFromONNXFormat(
```

Comment:
This part is also duplicated with `DequantizeOpWeights`, there are some more differences, but maybe it is still better to separate the code and throw the common code into separate functions.

Reply:
Done